Makefile

```diff
@@ -6,5 +6,11 @@ check-licence:
 		find . -name "*.go") | sed -e 's,\./,,' | sort | uniq -u | \
 		xargs -I {} echo FAIL: licence missed: {}
 
-check: check-licence
+check-go:
+ifneq ($(strip $(shell gofmt -l .)),)
+	$(error go fmt is sad)
```

Comment:
you might want to just follow the "verify.bash" model. Its actually really nice to be told what files gofmt is unhappy about.
Also, I don't think 'gofmt -l .' recurses into subdirectories.

I also wonder if we should be adding
.PHONY: check-go check-licence

(shouldn't it be spelled license ?)